docs/modules/sql/pages/user-defined-types.adoc

```diff
@@ -1,173 +1,161 @@
 = User-Defined Types in SQL
 
-User-Defined Types (also known as UDTs, nested types, or nested fields) is an experimental feature that allows you to create
- custom data types that can be referenced in the xref:sql:create-mapping.adoc[CREATE MAPPING statement].
-
-UDTs are useful for creating and accessing hierarchical data structures, including simple cases of nested objects and
-more complex cases of fully/partially connected graphs of objects. See <<queryingExamples,examples>> for more information.
-
-WARNING: Support for UDTs is an experimental feature, and disabled by default.
-See <<enabling-udt-support, Enabling UDT support>> to learn how to enable it.
+User-Defined Types (also known as UDTs, nested types, or nested fields) allow you to create custom data types that can be referenced in the xref:sql:create-mapping.adoc[CREATE MAPPING statement]. UDTs are also useful for creating and accessing hierarchical data structures.
 
 == Feature Overview
 
-Due to the experimental nature of UDTs, the feature set is limited to the following:
-
-- Support for three formats - `portable`, `compact` and `java` with varying level of sub-feature support
-- Instance and type-level cycles are only supported for Java types.
-- Type formats (referred to as type kind or kind hereafter) can not be mixed, both between mappings and types themselves.
-For example, you cannot use Java type in another Portable type or Portable mapping.
-- Limited support for instance and type-level cycles; only for Java types.
-Note that INSERT/UPDATE are disabled for mappings that use type hierarchies that contain cycles.
-- Limited support for `INSERT` and `UPDATE` queries for mappings using UDTs; if a `TYPE` hierarchy contains cycles,
-`INSERT` and `UPDATE` statements are disabled for the mapping.
-- Support for using UDT-based projections in both normal `SELECT` projection lists, `WHERE` filters and in `JOIN` conditions.
-
-== Enabling UDT Support
-You can enable UDTs by setting the `hazelcast.sql.experimental.custom.types.enabled` property to `true` in the member configuration.
-[tabs]
-==== 
-XML:: 
-+ 
--- 
-[source,xml]
-----
-<hazelcast>
-    <properties>
-        <property name="hazelcast.sql.experimental.custom.types.enabled">true</property>
-    </properties>
-</hazelcast>
-----
---
-
-YAML::
+- `portable`, `compact`, `java` and `avro` formats are supported.
+- Cyclic definitions are only supported for Java-serialized structures. Portable and Compact formats do not support cyclic schemas, and Avro support is currently limited to acyclic schemas.
 +
-[source,yaml]
-----
-hazelcast:
-  properties:
-    hazelcast.sql.experimental.custom.types.enabled: true
-----
-
-Java::
-+
-[source,java]
-----
-final Config config = new Config();
-config.setProperty("hazelcast.sql.experimental.custom.types.enabled", "true");
-----
-====
+NOTE: If mappings use cyclic structures, the `INSERT` and `UPDATE` queries are disabled.
```

Comment:
this is repeated later in "Support for Cycles". IMO this is too much detail for "feature overview". In this section we can mention that we support cycles but there are important limitations described in detail later.

Reply:
fef8c8c2